src/index.ts

```diff
@@ -21,7 +22,7 @@ const readFile = fs.readFile
       );
     };
 
-const GOOGLE_TOKEN_URL = 'https://www.googleapis.com/oauth2/v4/token';
+const GOOGLE_TOKEN_URL = 'https://oauth2.googleapis.com/token';
```

Comment:
I only touched the urls to be up to date with the docs, which seem to want us to use them... if we're scared about changing many things at once, this could be left out .